### PR TITLE
B 1862 add start your journey bloc wait until mock up is ready

### PR DESCRIPTION
**Pull request by pixelfact**

### CODE REVIEW

File: app/home/features/journey/journey.private.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import { IconTag } from "components/ds/icon-tag/icon-tag";
+import { ProgressBar } from "components/ds/progress-bar/progress-bar";
+import { BaseLink } from "components/layout/base-link/base-link";
+import { Icon } from "components/layout/icon/icon";
+import { Section } from "components/layout/section/section";
+import { Typography } from "components/layout/typography/typography";
+
+import { TJourney } from "./journey.types";
+
+function JourneyItem({ stepName, completion }: TJourney.JourneyItemProps) {
```
Comment (alexbeno) [inline app/home/features/journey/journey.private.tsx:20]: completed ?

File: app/home/features/journey/journey.public.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+export function JourneyPublic(_: TJourney.JourneyPublicProps) {
+  const { loginWithRedirect } = useAuth0();
+  const { T } = useIntl();
+  function handleLogin() {
+    handleLoginWithRedirect(loginWithRedirect);
+  }
+
+  const blogUrl =
+    "https://blog.onlydust.com/how-starknet-can-help-provide-real-world-solutions-starting-with-decentralized-financial-solutions/";
```
Comment (alexbeno) [inline app/home/features/journey/journey.public.tsx:33]: we should store that in translate or env variable ?

File: app/home/features/journey/journey.hooks.ts [hunk 3]
```diff
@@ -1,3 +1,10 @@
+import { meApiClient } from "api-client/resources/me";
+
 export function useJourney() {
-  return true;
+  const { data, isLoading, isError } = meApiClient.queries.useGetMyJourney({});
+  return {
+    isLoading,
+    isError,
+    completed: data?.completed,
```
Comment (haydencleary) [inline app/home/features/journey/journey.hooks.ts:8]:
```suggestion
    completed: data?.completed ?? false,
```

File: app/home/features/journey/journey.public.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+          </>
+        }
+        classNames={{
+          section: "hidden md:flex",
+        }}
+      >
+        <Card background={"base"} className="flex items-center gap-6">
+          <Image
+            src={odCommunityTips}
+            alt="Onlydust community tips"
```
Comment (haydencleary) [inline app/home/features/journey/journey.public.tsx:92]: Trad ?

File: app/home/features/journey/journey.types.ts [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
 
 export namespace TJourney {
-  export interface Props extends PropsWithChildren {}
+  export interface JourneyItemProps {
+    stepName: string;
+    completion: boolean;
+  }
+  export interface JourneyPrivateProps {}
+
+  export interface JourneyPublicProps {}
```
Comment (haydencleary) [inline app/home/features/journey/journey.types.ts:12]: Can remove ?

File: app/home/page.tsx [hunk 6]
```diff
@@ -27,12 +29,13 @@ export default function HomePage() {
...
-            <Journey />
-          </JourneyGuard>
-
           <RequiredAuthGuard>
             <Profile />
+            <ErrorBoundary fallback={null}>
+              <Suspense fallback={<JourneyPrivateLoading />}>
+                <JourneyPrivate />
+              </Suspense>
+            </ErrorBoundary>
```
Comment (haydencleary) [inline app/home/page.tsx:38]: No suspense here

File: app/home/page.tsx [hunk 7]
```diff
@@ -27,12 +29,13 @@ export default function HomePage() {
       <Container>
         <PosthogOnMount eventName={"home_dashboard_viewed"} />
         <PageGrid>
-          <JourneyGuard>
-            <Journey />
-          </JourneyGuard>
-
           <RequiredAuthGuard>
```
Comment (haydencleary) [inline app/home/page.tsx:32]: Need fallback skeleton

File: translations/v2/en/pages/home/journey.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+      "title": "Complete your first contribution.",
+      "description": "Even if it’s not rewarded, we value your work on your profile."
+    },
+    "step3": {
+      "title": "Claim your first reward",
+      "description": "Go to your reward page and follow the 2 min process."
+    },
+    "step4": {
+      "title": "Update your description",
+      "description": "To help other find out who you really are."
```
Comment (haydencleary) [inline translations/v2/en/pages/home/journey.json:19]:
```suggestion
      "description": "To help others find out who you really are."
```

File: translations/v2/en/pages/home/journey.json [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "step5": {
+      "title": "Update your contact methods.",
+      "description": "Make sure that you’re reachable through multiple accounts."
+    }
+  },
+  "public": {
+    "subtitle": "Sign-up and create your account",
+    "description": "Get access to more than 500 projects to contribute."
```
Comment (haydencleary) [inline translations/v2/en/pages/home/journey.json:28]:
```suggestion
    "description": "Get access to more than 500 projects to contribute to."
```

**Review by haydencleary — APPROVED** [review]

**Pull request merged**